Dockerfile

```diff
@@ -1,9 +1,7 @@
-FROM nicks/plovr-deps
+FROM xvilo/plovr-base
```

Comment:
wait, where does this new docker image come from? how do we build it?

Reply:
Hi @nicks, 

Sorry for the late response, totally forgot about it. It's the `Dockerfile.base` with some extra personal additions. I have reverted this and update both Docker files a bit too! 

Let me know 👍